cpp/include/cudf/detail/null_mask.cuh

```diff
@@ -141,6 +142,285 @@ void inplace_bitmask_binop(
   stream.synchronize();
 }
 
+namespace {
```

Comment:
Don't use anonymous namespaces in headers: https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file

Reply:
@jrhemstad @nvdbaranec JFYI this PR looks like it got merged without addressing this. It's not a big deal but may be worth fixing in a follow-up to avoid the possibility of unexpected UB somewhere down the line.